transaction/payload.go

```diff
@@ -8,8 +8,7 @@ import (
 )
 
 const (
-	SubscriptionsLimit      = 1000
-	BucketsLimit            = 1000
+	SubscriptionsLimit      = 100000
 	MaxSubscriptionDuration = 65535
```

Comment:
What about doubling MaxSubscriptionDuration? I feel that the current limitation is a bit too short based on a few feedbacks

Reply:
generally I'm not against that, but I feel like they think about it being too short because you can't extend existing subscription and you need to wait for it to end first. since that isn't the case after this PR is merged, just extending it every once in a while should be sufficient.

current max duration is about ~18.2 days